RELEASE-NOTES.md

```diff
@@ -77,6 +77,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Inlet: Fixed a bug relating to nested table lookups of primitive arrays and functions
 - Fixed a bug relating to deeply nested callback functions in Inlet
 - Inlet: Always ignore primitive array elements that do not match the requested type
+- Inlet: Empty structs/collections of structs with required sub-elements no longer fail
```

Comment:
"Optional empty structs/collections of structs with required sub-elements no longer fail"?

Reply:
`verification` is on the next line - I think it renders properly: https://github.com/LLNL/axom/blob/2bbab0e52f1d7964bc70e6be13af094aca3219e1/RELEASE-NOTES.md

Reply:
Sorry i was refering to adding "Optional" explicitly.